### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by Noppy-Kids** [termux/termux-packages#16780] [Q&A] [answered]
Can't installed using pkg & apt
![Screenshot_20230528_081853.jpg](https://github.com/termux/termux-packages/assets/134784403/44429d66-c979-4dc8-b04f-3ce956fbb9f1)

How that? Pls help i am new

**Comment by sylirre** [comment, marked as answer]
First of all, never ever use Termux from Play Store. It is deprecated and will not be updated anymore. *And for those who wonder why it still present on Play Store - it **can't** be deleted as we don't have Play Store Console access*.

https://github.com/termux/termux-app#installation

Get the Termux from F-Droid: https://f-droid.org/packages/com.termux/

If on Android OS versions 5 or 6: https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6

Also don't run `apt` or `pkg` as root. This is not supported as will mess up environment. There is added a [special patch](https://github.com/termux/termux-packages/blob/5cc582efb1116d305a491adbb790fabb636c0554/packages/apt/0010-prevent-usage-as-root.patch)  preventing execution of `apt` under root user. You will see a message *"Ability to run this command as root has been disabled permanently for safety purposes."*.
